### ISSUE TITLE
Shall README provide example of a "query string" or link to some RFC that describes them?

### ISSUE THREAD

**Issue by vi** [samscott89/serde_qs#104]
The "URL" in "[serde_urlencoded](https://github.com/nox/serde_urlencoded)" in the README suggests it is related to the web/browsers/HTTP world, but it is not obvious from README or repository description. I think README should provide more info about the format being parsed.

Compare with [README of the linked library](https://github.com/nox/serde_urlencoded/blob/master/README.md) - it contains a link to a [specification-esque document](https://url.spec.whatwg.org/#application/x-www-form-urlencoded) (but no example though) and a specific content type.

**Comment by samscott89** [comment]
Hey @vi. It's a very reasonable request.

I _think_ the crate documentation does a better job at describing the purpose: https://docs.rs/serde_qs/latest/serde_qs/

What do you think? Are there parts of that you think I should reuse for the README?

**Comment by vi** [comment]
Yes, that code example looks on the point. Something like this, but shorter and simpler (i.e. without nesting, only with e.g. two parameters), unless those square brackets and nesting is a distinctive advantage of the crate.

A link to RFC formally describing query strings would also be welcome (in crate documentation too).
